package.json

```diff
@@ -7,21 +7,21 @@
     "src",
     "public",
     "@types",
-    "ipfs-core-types",
     "docs",
     "build"
   ],
   "scripts": {
     "start": "run-script-os",
     "start:win32": "@powershell -Command $env:REACT_APP_GIT_REV=(git rev-parse --short HEAD); react-app-rewired-esm start",
     "start:darwin:linux": "cross-env NODE_ENV=development REACT_APP_ENV=development REACT_APP_GIT_REV=`git rev-parse --short HEAD` react-app-rewired-esm start",
-    "lint": "run-s eslint typecheck",
+    "lint": "run-s eslint typecheck dep-check",
```

Comment:
this isn't called by CI, but probably should be. will address later (combine workflows currently doing typecheck, eslint, & depcheck separately)